trace-utils/src/send_data/mod.rs

```diff
@@ -853,17 +889,119 @@ mod tests {
             &Endpoint {
                 api_key: None,
                 url: "http://127.0.0.1:4321/".parse::<hyper::Uri>().unwrap(),
+                timeout: MILLIS_PER_SECOND,
             },
         );
 
         let res = data.send().await;
 
         assert!(res.last_result.is_err());
-        assert_eq!(res.errors_timeout, 0);
-        assert_eq!(res.errors_network, 1);
+        match std::env::consts::OS {
```

Comment:
Why do we expect a timeout for windows instead of a network error ?

Reply:
Oh, that drove me crazy. I'm not really familiar with Windows TCP/IP stack but it seems that, at hyper level, it returns a timeout rather than a connection refused error despite not having a listening socket on that port (4321).

Reply:
Why am I not surprised 😩. Maybe add a comment to avoid confusion